chapters/buffer_device_address.adoc

```diff
@@ -0,0 +1,181 @@
+// Copyright 2024 The Khronos Group, Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+// Required for both single-page and combined guide xrefs to work
+ifndef::chapters[:chapters:]
+ifndef::images[:images: images/]
+
+[[buffer-device-address]]
+= Buffer Device Address
+
+Buffer Device Address allows you to have a pointer to the `VkBuffer` in your shaders.
+
+[NOTE]
+====
+This will be more of a technical breakdown of how it works, if you looking for a simple "how do I just use this", then please take a look at the link:https://github.com/KhronosGroup/Vulkan-Samples/tree/main/samples/extensions/buffer_device_address[Vulkan Samples].
+====
+
+== Extension
+
+The original proposal was done with the  `VK_EXT_buffer_device_address` extension. Shortly afterwards, `VK_KHR_buffer_device_address` was added, which had some minor feature differences. Starting in Vulkan 1.2 this has become core and it is very widely avaiable on GPUs on every platform. Starting in Vulkan 1.3 it became required, so if you are using Vulkan 1.3, you are guaranteed support.
+
+=== Name alias
+
+The naming of this `Buffer Device Address` feature will be different depending on where you look.
+
+For GLSL, it is `buffer reference` (see link:https://github.com/KhronosGroup/GLSL/blob/main/extensions/ext/GLSL_EXT_buffer_reference.txt[GL_EXT_buffer_reference]).
```

Comment:
Maybe say "because syntactically it's used more like a C++ reference than a pointer"